web/cobrands/sass/_base.scss

```diff
@@ -366,6 +366,11 @@ legend,
   }
 }
 
+// For GOVUK Styling
+.govuk-radios__label {
```

Comment:
I agree this needs to be overridden, it probably should be in _gov.scss, though, in that case?